openmetadata-ui/src/main/resources/ui/cypress/common/Utils/Tags.ts

```diff
@@ -69,19 +81,25 @@ export const removeTags = (
 ) => {
   const tags = Array.isArray(inputTag) ? inputTag : [inputTag];
   interceptURL('PATCH', `/api/v1/${endPoint}/*`, 'removeTags');
+  interceptURL(
+    'GET',
+    `/api/v1/search/query?*index=tag_search_index*`,
+    'searchTags'
+  );
   tags.forEach((tag) => {
     cy.get(
       '[data-testid="entity-right-panel"]  [data-testid="tags-container"] [data-testid="edit-button"]'
     )
       .scrollIntoView()
       .click();
+    verifyResponseStatusCode('@searchTags', 200);
 
     // Remove all added tags
-    cy.get(`[data-testid="selected-tag-${tag}"] [data-testid="remove-tags"]`)
-      .scrollIntoView()
-      .click();
+    cy.get(
+      `[data-testid="selected-tag-${tag}"] [data-testid="remove-tags"]`
```

Comment:
I have seen some failure around here, Any reason we are removing scrollIntoView here?

Reply:
I believe adding scrollIntoView might be shifting the position of the Update button, which could be the cause of the issue. However, I'm still investigating to pinpoint the root cause, as it's only failing in CI and not locally.